### ISSUE TITLE
AUR breaks when git lfs is used for res/background.png

### ISSUE THREAD

**Issue by silverhadch** [pop-os/cosmic-greeter#69]
Its not possible to build the greeter for cosmic because git lfs is not supported throug Arch

**Comment by mmstick** [comment]
You need to enable support for it

**Comment by silverhadch** [comment]
Yeah i know use the makepkg-git-lfs-proto tool. I also made a pull request in the main repo fixing the Arch Setup Instructions. And enabling lfs on the start of the build. Just have to wait to being merged.
